retriever/bin/retriever.js

```diff
@@ -124,6 +123,7 @@ export default {
       )
       const response = new Response(originResponse.body, originResponse)
       setContentSecurityPolicy(response)
+      setOriginUrl(response, originUrl)
```

Comment:
Hmm, maybe this name is a bit confusing. Would the following be easier to understand?


```js
setOriginUrlHeader(response, originUrl)
```

Reply:
Or let's just inline this one-liner? That's simplest to me